Avoid hardcoding a `.so` extension and let GLib figure it out.
--- libplanner/mrp-storage-module-factory.c.orig	2023-12-17 00:31:55
+++ libplanner/mrp-storage-module-factory.c	2024-05-24 03:28:43
@@ -110,7 +110,7 @@
 	fullname = g_strconcat ("storage-", name, NULL);
 
 	path = mrp_paths_get_storagemodule_dir (NULL);
-	libname = g_module_build_path (path, fullname);
+	libname = g_build_filename (path, fullname, NULL);
 	g_free (path);
 
 	if (!module_hash) {
